accounts-journey/presentation/ui/account_detail/AccountDetailsViewController.swift

```diff
@@ -0,0 +1,278 @@
+//
+//  AccountDetailsViewController.swift
+//  AccountsJourney
+//
+//  Created by Backbase R&D B.V. on 29/11/2023.
+//
+
+import UIKit
+import SnapKit
+import Combine
+import Resolver
+import BackbaseDesignSystem
+
+final class AccountDetailsViewController: UIViewController {
+    
+    // MARK: - Properties
+    var viewModel: AccountDetailsViewModel
+    private var arrangementId: String
+    private var cancellables = Set<AnyCancellable>()
```

Comment:
can be just one cancellable, since only one element is stored in it

Reply:
According to the documentation this has to be a Set of AnyCancellables
```swift
/// Stores this type-erasing cancellable instance in the specified set.
///
/// - Parameter set: The set in which to store this ``AnyCancellable``.
@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
final public func store(in set: inout Set<AnyCancellable>)
```